SEGSupport/CMakeLists.txt

```diff
@@ -1,29 +1,42 @@
 #-----------------------------------------------------------------------------
-set(MODULE_NAME SEG2NRRD)
+# Not used
+set(MODULE_NAME SEGSupport)
 
 #-----------------------------------------------------------------------------
 set(MODULE_INCLUDE_DIRECTORIES
   )
 
-set(MODULE_SRCS
-  SEG2NRRD.cxx
-  framesorter.h
-  SegmentAttributes.h
-  )
-
+#-----------------------------------------------------------------------------
 set(MODULE_TARGET_LIBRARIES
+  ${DCMTK_LIBRARIES}
   ${ITK_LIBRARIES}
-  )
+  xml2 z
```

Comment:
I see .. and the original commit is not helpful either. 
See QIICR/Iowa2DICOM@8302cae28c

@fedorov If this is an issue with building DCMTK, would be great to document it ... so that we can fix it. :smile: